test/expected.txt

```diff
@@ -227,6 +227,17 @@
 || 
 || 2 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  find Char]
+|| 	1. Charlie Dickson Email: charlie.d@nus.edu.sg Address: 333, gamma street Tags: [school][friends]
+|| 
+|| Did you mean to find these persons ?
+|| 1 persons listed!
+|| ===================================================
+|| Enter command: || [Command entered:  find Char Betsy]
+|| 	1. Betsy Choo Tags: [secretive]
```

Comment:
Shouldn't this return both Betsy Choo and Charlie Dickson?

Reply:
I have coded the substring method in a way such that if both keywords are substrings of the names then Betsy and Dickson will both be returned. If either one is a complete name, then the substring method would return the person with the complete name only. I have updated this in my user guide and I hope it doesnt confuse people >.<